_layouts/page-sidenav.html

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+---
+{% include _banner-title.html %}
+
+<div id="main" role="main">
+  <div class="ds-l-container ds-base ds-u-margin-bottom--7" id="page-with-sidenav">
+    <section aria-label="Side NavBar">
+      <ul class="ds-c-list ds-c-list--bare sidenav">
+        {% if page.id == "data" %}
+          {% for item in site.data_nav %}
+            <li class="sidenav__item">
+              <a href="#{{ item.id }}" class="ds-u-display--block sidenav__link">{{ item.title }}</a>
+            </li>
+          {% endfor %}
+        {% elsif page.id == "pilot" %}
+          {% for item in site.pilot_nav %}
+            <li class="sidenav__item">
+              <a href="#{{ item.id }}" class="ds-u-display--block sidenav__link">{{ item.title }}</a>
+
+              {% if item.subnav.size > 0 %}
+                <ul>
+                  {% for subitem in item.subnav %}
+                    <li class="sidenav__item">
+                      <a href="#{{ subitem.id }}" class="ds-u-display--block sidenav__link">{{subitem.title}}</a>
+                    </li>
+                  {% endfor %}
+                </ul>
+              {% endif %}
+            </li>
+          {% endfor %}
```

Comment:
This is the same logic repeated for specific pages. It also doesn't seem that it's temporary (e.g. until releasing the user guide). How can we DRY it up? Can we set up a hash of `page.id` => nav_items?